src/Aspire.ProjectTemplates/templates/aspire-starter/9.2/Aspire-StarterApplication.1.Tests/WebTests.cs

```diff
@@ -19,7 +19,17 @@ public class WebTests
     public async Task GetWebResourceRootReturnsOkStatusCode()
     {
         // Arrange
-        var appHost = await DistributedApplicationTestingBuilder.CreateAsync<Projects.GeneratedClassNamePrefix_AppHost>();
+#if (TestFx == "MSTest")
+        CancellationToken cancellationToken = new CancellationTokenSource(DefaultTimeout).Token;
+#elif (TestFx == "NUnit")
+        CancellationToken cancellationToken = TestContext.CurrentContext.CancellationToken;
```

Comment:
The `DefaultTimeout` member isn't being used in this case right, and the xUnit.net v3 case. We shouldn't emit it if it's not used.

Reply:
Ah I see now, you're passing it to `WaitAsync` calls, along with the token. Seems a bit redundant in those cases but I guess it's fine?

Reply:
it is redundant given there's an overload for just ct, but yet more #if's would make this even harder to read. Plus, in both nunit and xunit v2 one can get a proper cancelation token from the test runner (using property) as mentioned, and if the user adds that, it might not be redundant (as you might want both)

So leaving as is unless you feel strongly